celery_singleton/inspect_celery.py

```diff
@@ -0,0 +1,35 @@
+from celery import Celery
+
+class CeleryTask:
+    def __init__(self, name, args, kwargs):
+        self.name = name
+        self.args = args
+        self.kwargs = kwargs
+
+def list_celery_tasks_with_name(app: Celery, name: str) -> [CeleryTask]:
```

Comment:
[CeleryTask] is not a valid type hint. Also please don't use type hints for now, we'll leave that for another release. They're not used anywhere else and we're technically still supporting python3.4

Reply:
Thanks for reminding me ! I am removing type hints